lib/mix/tasks/phx.gen.notifier.ex

```diff
@@ -0,0 +1,167 @@
+defmodule Mix.Tasks.Phx.Gen.Notifier do
+  @shortdoc "Generates a notifier to send emails using the mailer"
+
+  @moduledoc """
+  Generates a notifier to send emails using the mailer
```

Comment:
This description is technically correct but it reads weird. It feels "notifier" is an empty word here and maybe we should go with `mix phx.gen.email` but I am not sure. The templates literally import `Swoosh.Email` though.